src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 endpoint.userSearch=/v1/users/uuidBySlack
-user.baseURL=https://users
\ No newline at end of file
+endpoint.addKeeper=/keepers
```

Comment:
it is not correct url because you will add it to baseUrl and will receive `http://keepers/keepers`